src/cache.rs

```diff
@@ -143,40 +144,65 @@ impl Cache {
     }
 
     /// Search for a page and return the path to it.
-    pub fn find_page(&self, name: &str) -> Option<PathBuf> {
-        // Build page file name
-        let page_filename = format!("{}.md", name);
-
+    pub fn find_pages(&self, name: &str, config: &Config) -> Vec<PathBuf> {
         // Get platform dir
         let platforms_dir = match Self::get_cache_dir() {
             Ok(cache_dir) => cache_dir.join("tldr-master").join("pages"),
             Err(e) => {
                 log::error!("Could not get cache directory: {}", e);
-                return None;
+                return Vec::new();
             }
         };
 
-        // Determine platform
-        let platform = self.get_platform_dir();
+        // Get custom pages directory
+        let custom_pages_directory = &config.directories.custom_pages_dir;
 
-        // Search for the page in the platform specific directory
-        if let Some(pf) = platform {
-            let path = platforms_dir.join(&pf).join(&page_filename);
-            if path.exists() && path.is_file() {
-                return Some(path);
-            }
+        // Look up custom page (<name>.page). If it exists, return it directly,
+        let custom_page = format!("{}.page", name);
```

Comment:
Can you inline this variable? I prefer to avoid shadowing variables with different types 🙂  Simply use `.join(&format!("{}.page", name))`.